librz/arch/p/asm/asm_hexagon.c

```diff
@@ -194,6 +194,8 @@ RzAsmPlugin rz_asm_plugin_hexagon = {
 	.arch = "hexagon",
 	.author = "Rot127",
 	.license = "LGPL3",
+	.cpus = "v79 (LLVM based)",
```

Comment:
```suggestion
	.cpus = "v79",
```

Reply:
Nah, I added it intentionally. Because the LLVM version misses some instructions (e.g. some v79 HVX instructions).

Once I finally manage to finish https://github.com/rizinorg/rz-hexagon/issues/84 it can be removed. Because the official SDK contains also undocumented instructions.

Reply:
Cannot be there tho

Reply:
Maybe say v79llvm